docs/sources/enterprise/_index.md

```diff
@@ -6,75 +6,73 @@ type = "docs"
 [menu.docs]
 name = "Grafana Enterprise"
 identifier = "enterprise"
-weight = 30
+weight = 100
 +++
 
 # Grafana Enterprise
 
-Grafana Enterprise is a commercial edition of Grafana that includes additional features not found in the open source
-version.
+Grafana Enterprise is a commercial edition of Grafana that includes additional features not found in the open source version.
 
-Building on everything you already know and love about Grafana, Grafana Enterprise adds enterprise data sources,
-advanced authentication options, more permission controls, 24x7x365 support, and training from the core Grafana team.
+Building on everything you already know and love about Grafana, Grafana Enterprise adds enterprise data sources, advanced authentication options, more permission controls, 24x7x365 support, and training from the core Grafana team.
 
 Grafana Enterprise includes all of the features found in the open source edition and more.
 
-___
+[Learn more about Grafana Enterprise.](https://grafana.com/enterprise)
 
-### Enhanced LDAP Integration
+## Enhanced security features
 
-With Grafana Enterprise you can set up synchronization between LDAP Groups and Teams. [Learn More]({{< relref "../auth/enhanced_ldap.md" >}}).
+Grafana Enterprise includes high-level security integration.
```

Comment:
Really appreciate the grouping of the auth(entication|orization) functionalities.

Since this looks like a new addition to me: High-level as not near the core or high-level as in fancy? I don’t recognize high-level as a way to describe security features. Proposing an alternative here if you like it (but this is definitely not a blocker and it might very well be me lacking knowledge about this particular term).

```suggestion
Grafana Enterprise includes integrations with more ways to authenticate your users and enhanced authorization capabilities.
```